packages/schema/lib/schemas/SearchOrCreateSchema.js

```diff
@@ -30,6 +32,21 @@ module.exports = makeSchema(
         description: 'The key of the create that powers this search-or-create',
         $ref: KeySchema.id,
       },
+      update: {
+        description:
+          'The key of the create that will be used if a search succeeds; having a value here converts this search-or-create into a search-and-create',
```

Comment:
Let's add "EXPERIMENTAL" to the help text to discourage external partners from using it until it gets stable.

```suggestion
          'EXPERIMENTAL: The key of the create that will be used if a search succeeds; having a value here converts this search-or-create into a search-and-create',
```

Same goes for `updateInputFromSearchOutput` and `searchUniqueInputToOutputConstraint`.